### PR TITLE
Fixes series name encoding issues

### PR DESCRIPTION
**Pull request by liiight**
Fixes series name encoding issues.
Issue #3129

### CODE REVIEW

File: flexget/plugins/estimators/est_release_series_tvmaze.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ class EstimatesSeriesTVMaze(object):
     def estimate(self, entry):
         if not all(field in entry for field in ['series_name', 'series_season', 'series_episode']):
             return
-        series_name = entry['series_name']
+        series_name = entry['series_name'].encode('utf-8')
```
Comment (gazpachoking) [inline flexget/plugins/estimators/est_release_series_tvmaze.py]: All entry fields should be unicode strings, not byte strings.

Comment (gazpachoking) [inline flexget/plugins/estimators/est_release_series_tvmaze.py]: Hmm, I read this backwards maybe jumped the gun with my comment. I'll keep looking.

Comment (liiight) [inline flexget/plugins/estimators/est_release_series_tvmaze.py]: I'm not encoding an entry field, just a local variable used for logging.

**Comment by liiight** [comment]
Fixed on pytvmaze side, this is no longer required

**Comment by liiight** [comment]
Fixed on pytvmaze side, this is no longer required

**Pull request closed without merging**